test/emitters.js

```diff
@@ -284,7 +284,7 @@ test(`emitting dom api destroy`, t => {
 			cb(null, originalDomApi)
 		},
 		reset(context, cb) {
-			cb(null)
+			t.ok(false, `Reset should not be called`)
```

Comment:
Any reason you did `t.fail` above and `t.ok` here? 🤔

Reply:
hah no, just forgot about this one